lib/ipfs/IpfsStorage.ts

```diff
@@ -15,21 +15,21 @@ export default class IpfsStorage {
   /**
    * Static method to have a single instance of class and mock in unit tests
    */
-  public static create (repo?: any): IpfsStorage {
+  public static async create (repo?: any): Promise<IpfsStorage> {
```

Comment:
Hmm I think we should treat it as a singleton. When I was writing tests, I accidentally created multiple instances of IPFS and the library didn't like that. I can implement something like

`If instance exists, log message and print out current repo, then return existing instance`

Reply:
If we want this to be an instance then we should remove the parameter right? Otherwise, for each parameter input, we need to create and save that instance (which doesn't make sense to me but maybe it does .. ?)